src/components/layout.js

```diff
@@ -17,16 +17,20 @@ const Layout = ({ home, children, location }) => (
     <main
       sx={{
         position: `relative`,
-        backgroundColor: home ? `background` : `#eee`,
+        // backgroundColor: home ? `background` : `accentBackground`,
         pt: [`0`, `3`],
         pb: [`0`, `3`],
+        backgroundColor: `accentBackground`,
       }}
     >
       <Container
         sx={{
-          backgroundColor: `background`,
-          borderRadius: `2`,
-          py: `4`,
+          borderRadius: [`0`, `2`],
```

Comment:
Do we need the border radius anymore? Since it's all the same background color, we don't see the effect of the border radius